### DISCUSSION TITLE
Customizable profiles for workspaces within the same window, similar to Arc.

### DISCUSSION THREAD

**Discussion by IAmTahazzot** [zen-browser/desktop#3820] [Polls]
Adding customizable profiles for each workspace within the same window, similar to Arc, would be a game-changer. If no profile is set, the default will apply, but users can assign one while creating a workspace.

Seamlessly switching profiles without opening new windows keeps everything streamlined and efficient. For `Zen`, this feature perfectly reflects the vision:

>  "Efficiency without compromise."

**Comment by giaaaacomo** [comment]
isn't this already available through containers? you can set a default container per workspace and into that you can log with whatever account you'd like, not affected by what are you logged into outside the container. and if i'm not wrong, you can have different containers per workspace
